### DISCUSSION TITLE
Tin details

### DISCUSSION THREAD

**Discussion by Nabins** [CMSgov/price-transparency-guide#652] [General]
For any EIN, is there any way to find its details (example:  Hospital's Name, address, phone numbers)?? .

**Comment by tylerrollinsphd** [comment]
i would like to know this too...the rates don't mean much if we don't know where the practitioner is located for that rate, given that many providers work at multiple locations

**Reply by ardent-zph** [reply]
this would be very helpful addition

**Comment by shaselton-usds** [comment]
@Nabins @tylerrollinsphd @ardent-zph this looks like very much related to @ardent-zph's discussion #803. We are looking into how we can make the TIN more actionable.
